src/ledger/balanceSheet.ts

```diff
@@ -23,7 +20,7 @@ export const updateUsersPendingPayment = async () => {
   }
 }
 
-export const getBalanceSheet = async () => {    
+export const getLedgerAccounts = async () => {    
```

Comment:
I think `getBalanceSheet` was a more apt name for this function than `getLedgerAccounts`

Reply:
from an accounting point of view, balance sheet is really about liability + assets. here this function is returning some other stuff, that I think is best describe as ledger: https://www.freshbooks.com/hub/accounting/what-is-a-ledger